frontend/src/scenes/session-recordings/playlist/sessionRecordingsListPropertiesLogic.ts

```diff
@@ -15,41 +16,57 @@ export const sessionRecordingsListPropertiesLogic = kea<sessionRecordingsListPro
     })),
 
     actions({
-        loadPropertiesForSessions: (ids: string[]) => ({ ids }),
-        maybeLoadPropertiesForSessions: (ids: string[]) => ({ ids }),
+        loadPropertiesForSessions: (sessions: SessionRecordingType[]) => ({ sessions }),
+        maybeLoadPropertiesForSessions: (sessions: SessionRecordingType[]) => ({ sessions }),
     }),
 
     loaders(({ actions }) => ({
         recordingProperties: [
             [] as SessionRecordingPropertiesType[],
             {
-                loadPropertiesForSessions: async ({ ids }, breakpoint) => {
-                    const paramsDict = {
-                        session_ids: ids,
-                    }
-                    const params = toParams(paramsDict)
+                loadPropertiesForSessions: async ({ sessions }, breakpoint) => {
                     await breakpoint(100)
 
                     const startTime = performance.now()
-                    const response = await api.recordings.listProperties(params)
+                    const sessionIds = sessions.map((x) => `'${x.id}'`).join(',')
+
+                    const oldestTimestamp = sessions.map((x) => x.start_time).sort()[0]
+                    const newestTimestamp = sessions.map((x) => x.end_time).sort()[sessions.length - 1]
+
+                    const hogql: HogQLQuery = {
+                        kind: NodeKind.HogQLQuery,
+                        query: `SELECT properties.$session_id as session_id, any(properties) as properties
```

Comment:
I think this selects the first properties that ClickHouse evaluates, is that what you wanted?

Should it be `groupArray(properties)` to get all properties?

Reply:
Yep. We can only display one set of properties in the list anyways so I just want an event from the recording, the properties that matter should be the same anyways (OS, browser)